File: TwoSum/TwoSum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    int i, j;
    int *newnums = (int*)malloc(2 * sizeof(int));
    if ((nums == NULL) || (numsSize < 2))
    {
        return NULL;
    }
    
    for(int i = 0; i < numsSize-1; i++){
        for(int j = i + 1; j < numsSize; j++){
            if(nums[j] == target - nums[i]){
                newnums[0] = i;
                newnums[1] = j;
                return newnums;
            }
        }
    }
    
    return NULL;
}

File: TwoSum/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        length = len(nums)
        result = []
        for i in range(0,length-1):
            for j in range(i+1,length):
                tSum = nums[i] + nums[j]
                if tSum == target: 
                    result.append(i)
                    result.append(j)
                    return result
        
